routes/doc.tsx

```diff
@@ -150,6 +150,15 @@ export const pathGetHead = async <R extends DocRoutes>(
   ctx: RouterContext<R>,
 ) => {
   let { proto, host, item, path } = ctx.params;
+  if (proto === "deno") {
+    ctx.response.status = Status.MovedPermanently;
+    if (host === "unstable") {
+      ctx.response.redirect("https://deno.land/api");
+    } else {
+      ctx.response.redirect("https://deno.land/api?unstable");
+    }
+    return;
```

Comment:
why `return;`?

Reply:
because we dont want any of the code in the other branch to be run if we are redirecting

Reply:
Just for unambiguity, I would combine the return with the previous line.  It isn't specifically required, but it should raise less questions for those reading the code.